guide/controller-plugin.md

```diff
@@ -210,9 +210,9 @@ motor position range are written in red.
 
 ### Remote-control Plugin
 
-A remote-control plugin allow to simply and efficiently create an interface
+A remote-control plugin allows to simply and efficiently create an interface
 using the Webots API to communicate with a real robot. The main purpose of a
-remote-control library is to wrap all the Webots API functions used by the robot
+remote-controled library is to wrap all the Webots API functions used by the robot
```

Comment:
I believe this name should remain remote-control library, as "remote-control" is the name of the library.